modin/experimental/backends/omnisci/query_compiler.py

```diff
@@ -306,10 +306,28 @@ def groupby_agg(
         groupby_kwargs,
         drop=False,
     ):
+        def default_to_pandas():
+            return super(DFAlgQueryCompiler, self).groupby_agg(
+                by,
+                is_multi_by,
+                axis,
+                agg_func,
+                agg_args,
+                agg_kwargs,
+                groupby_kwargs,
+                drop,
+            )
+
         # TODO: handle `is_multi_by`, `agg_args`, `drop` args
-        new_frame = self._modin_frame.groupby_agg(
-            by, axis, agg_func, groupby_kwargs, **agg_kwargs
-        )
+        if callable(agg_func):
+            return default_to_pandas()
+        try:
+            new_frame = self._modin_frame.groupby_agg(
+                by, axis, agg_func, groupby_kwargs, **agg_kwargs
+            )
+        # Occurs when the aggregation function name wasn't found in OmniSci' functions dictionary
+        except NotImplementedError:
```

Comment:
I think we should handle this in `build_method_wrapper` to cover all functions at once (and then can remove similar code from `setitem` and `insert`).